modal/cls.py

```diff
@@ -421,7 +425,7 @@ class _Cls(_Object, type_prefix="cs"):
     """
 
     _class_service_function: Optional[_Function]  # The _Function (read "service") serving *all* methods of the class
-    _options: Optional[_ServiceOptions]
+    _options: _ServiceOptions
```

Comment:
I think it used to be the case that we used the presence of `_options` to determine if we needed to load an instance service function (FunctionBindParams) or not - i.e. if we had to do `FunctionBindParams` or could just use the class service function. I *think* we have tests for this "optimization" but could be worth double checking